README.md

```diff
@@ -55,9 +55,38 @@ var jobDetails = client.getJobDetails(job.id);
 `jobDetails` will contain all information normally found in a successful response from
 our [Get Job](https://www.rev.ai/docs#operation/GetJobById) endpoint
 
+### Checking multiple files
+
+You can retrieve a list of transcription jobs with optional parameters
+
+```javascript
+var jobs = client.getListOfJobs();
```

Comment:
I had mentioned this to @kbridbur - I think these (and other examples in the README) should be `await client.[whatever])`. since the methods return promises